docs/upgrade/2.4.1_to_2.4.2.rst

```diff
@@ -0,0 +1,142 @@
+
+This will prevent the graphical updater from attempting to re-apply the failed
+update and has no bearing on future updates. You can now perform a manual
+update by running the following commands: ::
+
+  cd ~/Persistent/securedrop
+  git fetch --tags
+  gpg --keyserver hkps://keys.openpgp.org --recv-key \
+   "2359 E653 8C06 13E6 5295 5E6C 188E DD3B 7B22 E6A3"
+  git tag -v 2.4.2
+
+The output should include the following two lines: ::
+
+    gpg:                using RSA key 2359E6538C0613E652955E6C188EDD3B7B22E6A3
+    gpg: Good signature from "SecureDrop Release Signing Key <securedrop-release-key-2021@freedom.press>" [unknown]
+
+
+Please verify that each character of the fingerprint above matches what is
+on the screen of your workstation. A warning that the key is not certified
+is normal and expected. If the output includes the lines above, you can check
+out the new release: ::
+
+    git checkout 2.4.2
+
+.. important:: If you do see the warning "refname '2.4.2' is ambiguous" in the
+  output, we recommend that you contact us immediately at securedrop@freedom.press
+  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+Finally, run the following commands: ::
+
+  ./securedrop-admin setup
+  ./securedrop-admin tailsconfig
+
+Troubleshooting Kernel Issues
+-----------------------------
+SecureDrop 2.4.2 includes a kernel update on the *Application* and *Monitor
+Servers*, from version 5.15.26 to version 5.15.57. As with all kernel updates,
+we have extensively tested this update against
+:ref:`recommended hardware <Specific Hardware Recommendations>`.
+
+If you are running SecureDrop on hardware that is not officially supported, you
+may encounter compatibility issues with the new kernel. For example, the servers
+may not boot, or you may lose network connectivity. If this happens, you can
+temporarily downgrade to the previous kernel version.
+
+.. important::
+
+   To ensure continued secure operation of your SecureDrop instance, it is of
+   critical importance to resolve any compatibility issues with the new kernel
+   as quickly as possible. If you encounter problems with this update, please
+   get in touch with us urgently, so we can help you run the latest supported
+   kernel version.
+
+For information on how to downgrade to the previous kernel, and for additional
+troubleshooting information, please see our :doc:`Kernel Troubleshooting Guide <../kernel_troubleshooting>`.
+
+Upgrade from Tails 4 to Tails 5
+-------------------------------
+
+If you have not already done so, you must manually upgrade from the Tails 4 release
+series to the Tails 5 series.
+
+.. important::
+
+   You must upgrade your workstations to the latest version of SecureDrop by following
+   the steps above *before* upgrading to the Tails 5 series. You can verify the version
+   of SecureDrop by running ``git status`` in your ``~/Persistent/securedrop`` directory.
+   The output should include "HEAD detached at 2.4.1".
```

Comment:
We can bump the version to 2.4.2 here as well.

Reply:
Yes! Thank you @eloquence I completely missed this one.

Reply:
Fixed :white_check_mark: